definitions/npm/webpack_v5.x.x/flow_v0.104.x-/webpack_v5.x.x.js

```diff
@@ -10,6 +10,29 @@ declare module 'webpack' {
 
   declare type WebpackError = $WebpackError;
 
+  declare type WebpackLogger = {|
+    getChildLogger: (arg0: string | (() => string)) => WebpackLogger,
+    error(...args: any[]): void,
```

Comment:
I thinks spread params should rather use `$ReadOnlyArray`, but probably not big deal here.

Reply:
Great tip! Will start using this going forward